examples/03_datetime_encoder.py

```diff
@@ -75,31 +76,29 @@
 # .....................
 #
 # We will construct a |ColumnTransformer| in which we will encode
-# the city names with a |OneHotEncoder|, and the date
-# with a |DatetimeEncoder|.
+# the date with a |DatetimeEncoder|.
 #
 # During the instantiation of the |DatetimeEncoder|, we specify that we want
 # to extract the day of the week, and that we don't want to extract anything
-# finer than minutes. This is because we don't want to extract seconds and
-# lower units, as they are probably unimportant.
+# finer than hours. This is because we don't want to extract minutes, seconds
+# and lower units, as they are unimportant.
 
 from sklearn.compose import make_column_transformer
-from sklearn.preprocessing import OneHotEncoder
 
 from skrub import DatetimeEncoder
 
 encoder = make_column_transformer(
-    (OneHotEncoder(handle_unknown="ignore"), ["city"]),
-    (DatetimeEncoder(add_weekday=True, resolution="minute"), "date.utc"),
+    (DatetimeEncoder(add_weekday=True, resolution="hour"), "date"),
```

Comment:
I think we don't need `make_column_transformer` here. If the goal is to show what the DatetimeEncoder extracts by itself before diving into the pipeline stuff, we can do `DatetimeEncoder().fit_transform(X["date"])`